corelib/dynamicemb/dynamicemb/batched_dynamicemb_tables.py

```diff
@@ -237,7 +237,6 @@ def _export_matched_and_gather(
 
     search_offset = 0
     search_capacity = dynamic_table.capacity()
-    batch_size = batch_size if batch_size < search_capacity else search_capacity
```

Comment:
Why is this line removed? Will it cause error when search_capacity > batch_size?